router/server.go

```diff
@@ -103,7 +103,7 @@ func NewRouter(ctx context.Context, h models.HandlerInterface, port int) *Router
 	gMux.Handle("/api/grafana/scan", h.ProviderMiddleware(h.AuthMiddleware(h.SessionInjectorMiddleware(h.ScanGrafanaHandler))))
 
 	gMux.Handle("/api/prometheus/config", h.ProviderMiddleware(h.AuthMiddleware(h.SessionInjectorMiddleware(h.PrometheusConfigHandler)))).
-		Methods("POST", "DELETE")
+		Methods("GET", "POST", "DELETE")
```

Comment:
@piyushsingariya what do we need to do to keep the swagger definition and the REST API docs up to date when changes to the REST APIs are made?

Reply:
Just add the docs for new methods or endpoints.